File: src/main/java/fromjavatokotlin/FromJavaToKotlinDemo.java
package fromjavatokotlin;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@SpringBootApplication
@EnableConfigurationProperties
public class FromJavaToKotlinDemo {

  public static void main(String[] args) {
    SpringApplication.run(FromJavaToKotlinDemo.class);
  }
}

File: src/main/kotlin/fromjavatokotlin/FromJavaToKotlinDemo.kt
package fromjavatokotlin

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties

@SpringBootApplication
@EnableConfigurationProperties
class FromJavaToKotlinDemo

fun main() {
    SpringApplication.run(FromJavaToKotlinDemo::class.java)
}